router-tests/error_handling_test.go

```diff
@@ -601,6 +601,7 @@ func TestErrorPropagation(t *testing.T) {
 				cfg.Mode = config.SubgraphErrorPropagationModePassthrough
```

Comment:
Please add a test that validates the functionality of the new flag.

Reply:
Done